fastlane/appstoreres/jetpack_metadata/en-US/app_store_screenshot_1.txt

```diff
@@ -1,3 +1,3 @@
-Safer.
-Faster.
-More traffic.
\ No newline at end of file
+Create and
+manage your site
+on the go
```

Comment:
FYI: Updating the files in the `en-US` folder does no harm, but the source of truth is what is in `source`. If our tooling does it's job right, these files will be updated based on the `source` values next time the automation runs.

Apologies, we know the setup is not the clearest. There are plans on how to improve it, but they have lower priority than other intiatives.